test/modules/encoders/test_clip_text_encoder.py

```diff
@@ -24,89 +26,69 @@ def build_encoder(
             embedding_dim=embedding_dim,
             use_clip_init=True,
             context_length=context_length,
+            heads=heads,
         ):
             return CLIPTextEncoder(
                 embedding_dim=embedding_dim,
                 use_clip_init=use_clip_init,
                 context_length=context_length,
+                heads=heads,
             )
 
         return build_encoder, build_text
 
-    def test_initialization(self, start):
```

Comment:
Do you mean to remove this test or plan to modify it? This was originally proposed to validate the effect of enabling clip initialization.

cc @sophiazhi  @ebsmothers

Reply:
I chatted with @ankitade about this yesterday and seems this test is a bit flaky, making it difficult to make changes to other parts of CLIP code without triggering failures. Given that the specific initialization is not used except for the pretraining from the paper (which we are not doing), I think it's OK to remove.